### ISSUE TITLE
Colour contrast for search box is too low

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons-frontend#1141]
### Describe the problem and steps to reproduce it:

The colour contrast in the search bar mocks is too low according to http://leaverou.github.io/contrast-ratio/#rgb%28255%2C255%2C255%29-on-rgb%28208%2C77%2C43%29 -- even at max contrast (as in the mocks) it's not quite enough contrast, but it is especially problematic because I added a fade to non-`:focus`ed search results and that contrast is much too low. (See screencast below.)

@sarahmonster suggested simply going darker rather than lighter in the search box, but I'm tagging this UX needed to get their input.

![sep-26-2016 21-12-10](https://cloud.githubusercontent.com/assets/90871/18852039/8daa325c-8436-11e6-8264-ec51e2897acf.gif)

**Comment by pwalm** [comment]
Going to take a look at colour site-wide and make sure it complies with accessibility standards today. Mocks to follow.

**Comment by mstriemer** [comment]
Once https://github.com/mozilla/addons-frontend/pull/1268 and #1269 land this should be all fixed up.

**Comment by muffinresearch** [comment]
Closing this based on the previous comment https://github.com/mozilla/addons-frontend/issues/1141#issuecomment-255884566
